include/boost/numeric/ublas/tensor/storage.hpp

```diff
@@ -0,0 +1,52 @@
+//
+//  Copyright (c) 2018-2019, Cem Bassoy, cem.bassoy@gmail.com
+//
+//  Distributed under the Boost Software License, Version 1.0. (See
+//  accompanying file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+//  The authors gratefully acknowledge the support of
+//  Fraunhofer IOSB, Ettlingen, Germany
+//
+
+#ifndef BOOST_UBLAS_TENSOR_STORAGE_HPP
+#define BOOST_UBLAS_TENSOR_STORAGE_HPP
+
+#include <algorithm>
+#include <array>
+#include <functional>
+#include <initializer_list>
+#include <iostream>
+#include <numeric>
+#include <type_traits>
+#include <unordered_map>
+#include <vector>
```

Comment:
Could you reduce the number of headers here?

Reply:
It was due to the compressed map now I removed it.